vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2613,10 +2613,107 @@ def get_kv_cache_spec(self) -> dict[str, KVCacheSpec]:
             format. Layers that do not need KV cache are not included.
         """
 
-        block_size = self.vllm_config.cache_config.block_size
+        attn_block_size = self.vllm_config.cache_config.block_size
```

Comment:
Can you keep the code path for non-mamba model simple to help people that don't work on these models?
For example
```python
kv_cache_spec = self._get_kv_cache_spec()
if has_mamba and has_attn:
    self.update_kv_cache_spec_for_hybrid_attention_mamba()
```